packages/l10n/src/utils.ts

```diff
@@ -1,4 +1,11 @@
-export const mapLocaleToIntlLocale = locale => {
+type Currency = {
+  label: string;
+  symbol: string;
+};
+
+type Currencies = Record<string, Currency>;
```

Comment:
We should be able to import them from the other file.

Or we move those types (currency, countries, etc) into a shared `types.ts`